src/test/java/de/tum/in/www1/artemis/exercise/textexercise/TextExerciseUtilService.java

```diff
@@ -318,11 +438,26 @@ public TextPlagiarismResult createTextPlagiarismResultForExercise(Exercise exerc
         return plagiarismResultRepo.save(result);
     }
 
+    /**
+     * Creates a TextAssessmentEvent for the given parameters.
```

Comment:
```suggestion
     * Creates a TextAssessmentEvent with the given parameters.
```
I think with sounds better here.